Source/SwiftLintFramework/Rules/OperatorUsageWhitespaceRule.swift

```diff
@@ -61,18 +66,19 @@ public struct OperatorUsageWhitespaceRule: OptInRule, ConfigurationProviderRule
         let oneSpace = "[^\\S\\r\\n]" // to allow lines ending with operators to be valid
         let zeroSpaces = oneSpace + "{0}"
         let manySpaces = oneSpace + "{2,}"
-        let variableOrNumber = "\\b"
+        let leadingVariableOrNumber = "(?:\\b|\\))"
+        let trailingVariableOrNumber = "(?:\\b|\\()"
 
         let spaces = [(zeroSpaces, zeroSpaces), (oneSpace, manySpaces),
                       (manySpaces, oneSpace), (manySpaces, manySpaces)]
         let patterns = spaces.map { first, second in
-            variableOrNumber + first + operators + second + variableOrNumber
+            leadingVariableOrNumber + first + operators + second + trailingVariableOrNumber
         }
         let pattern = "(?:\(patterns.joined(separator: "|")))"
 
-        let genericPattern = "<.+?>"
-        let validRangePattern = variableOrNumber + zeroSpaces + rangePattern +
-            zeroSpaces + variableOrNumber
+        let genericPattern = "<(?:\(oneSpace)|\\S)+?>" // not using dot to avoid matching new line
```

Comment:
this still can prevent some violations if there's `>` on the same line, for example:
```swift
method(1 <<   (6), otherParam: 1 > 2)
```

🐛